examples/routes/routes.yaml

```
@@ -0,0 +1,8 @@
+apiVersion: eno.k8s.io/v1alpha1
+kind: Route
+metadata:
+  name: ipv4-subnet1-route
+spec:
+  Prefix: "192.168.0.2"
```

```suggestion
  Prefix: "192.168.2.0"
```
This is a not valid prefix if you combine it with a /24 mask. We need to fix this and also validate.